README.md

```diff
@@ -9,7 +9,7 @@ If you do not want to test or explore this version, please switch to the final 1
 
 ## Performance Test
 
-[📊 Speed Bench Mark](https://otp-performance.leonard.io/) We run the SpeedTest (included in the code) to measure the performance for every PR merged into OTP. The test uses a fixed Norwegian Netex data set and Grafana is used to display the results.
+[📊 Speed Benchmark](https://otp-performance.leonard.io/) We run the SpeedTest (included in the code) to measure the performance for every PR merged into OTP. The test uses a fixed Norwegian Netex data set and Grafana is used to display the results. Each run is listed under the [GitHub Actions](https://github.com/opentripplanner/OpenTripPlanner/actions/workflows/performance-test.yml). If you need to run the test locally you can [download the NeTex file](https://leonard.io/otp/rb_norway-aggregated-netex-2021-12-11.zip) from the CI Server. There is a [text file](test/ci-performance-test/travelSearch-expected-results.csv) with the expected results witch might need to be updated when OTP is changed and the test fails.
```

Comment:
It;s actually not the CI server but another one of mine. Not sure if that matters.

For completeness, it's worth mentioning that I also use a specific Geofabrik OSM extract from January 1st 2021: https://download.geofabrik.de/europe/norway-210101.osm.pbf

Reply:
I know, maybe we do not need to say where it comes from, but point out that it is the actual thing use. 

For the osm file, is that also available from the CI workflow ? If, so we can take it from there, then we do not need to update this doc when using a new one and we are 100% sure we have the same data?

Reply:
If you like I can do the same thing as I did with the Netex file and put it on my own server.